src/functions/schema.ts

```diff
@@ -45,6 +49,7 @@ function getAjv(oasVersion?: Optional<number>): AJV.Ajv {
   const ajvOpts: AJV.Options = {
     meta: true, // Add default meta schemas (draft 7 at the moment)
     schemaId: 'auto',
+    allErrors,
```

Comment:
@P0lip When would we want to not retrieve all the errors?

Reply:
The default is false, so I simply retained it.
We'll want to report all errors in most cases, I believe, but I didn't want to introduce that change as a part of this PR, as this would cause even more noise.

Reply:
Ok.